### DISCUSSION TITLE
Any way to implement v11 on Expo managed project

### DISCUSSION THREAD

**Discussion by AdzeB** [rnmapbox/maps#3107] [Q&A]
Good day

I was wondering if there's any way to implement v11 on an expo-managed project. I have been going through a pull request, and all I have seen requires you to modify the pods and gradle file. I wanted to ask how do I modify the two files?

**Comment by mfazekas** [comment]
@AdzeB pls see the config plugin, that what updates the Pod/Gradle files in managed projects.

https://github.com/rnmapbox/maps/blob/093d63dfcb68cd9b2704cd0417d1340f8e8c0622/plugin/src/withMapbox.ts#L38

One way to implement this would be adding `RNMapboxMapsUseV11` similarly how RNMapboxMapsVersion is implemented already

**Reply by AdzeB** [reply]
Ohh Thanks, this will only affect IOS and I will need to build dev client?

**Reply by AdzeB** [reply]
@mfazekas I got the following error whilst trying to build dev-client, the deploymentTarget for the app is set to 15.0, using 10.1.0-beta.11

<img width="995" alt="image" src="https://github.com/rnmapbox/maps/assets/59929333/b4de98cb-aa7c-43f4-b491-399cfe082fb7">

**Reply by mfazekas** [reply]
You can customise iOS deployment target version with https://docs.expo.dev/versions/latest/sdk/build-properties/#pluginconfigtype

**Reply by AdzeB** [reply]
<img width="221" alt="image" src="https://github.com/rnmapbox/maps/assets/59929333/871f2fa4-cb6c-4671-837a-d2917a10ebaa">
This is what I was referencing when I said the project deployment target is 15

**Reply by balloman** [reply]
@AdzeB I'm actually looking to create a PR that will allow for the config plugin to apply v11 and useframworks

**Reply by balloman** [reply]
@AdzeB @mfazekas PR up!

**Reply by AdzeB** [reply]
@balloman Thank you
